index.js

```diff
@@ -224,18 +225,25 @@ function setKillTimeout(kill, signal, options, killResult) {
 		return;
 	}
 
-	const forceKillAfter = Number.isInteger(options.forceKillAfter) ?
-		options.forceKillAfter :
-		5000;
-	setTimeout(() => kill('SIGKILL'), forceKillAfter).unref();
+	const timeout = getForceKillAfterTimeout(options);
+	setTimeout(() => kill('SIGKILL'), timeout).unref();
```

Comment:
I only use inline arrow functions if the return value is used, otherwise it makes the code harder to read.

```suggestion
	setTimeout(() => {
		kill('SIGKILL');
	}, timeout).unref();
```